node/sc/sub_bridge_handler.go

```diff
@@ -310,43 +315,53 @@ func (sbh *SubBridgeHandler) broadcastServiceChainTx() {
 	logger.Debug("broadcastServiceChainTx ServiceChainTxData", "len(txs)", len(txs), "len(peers)", len(peers))
 }
 
+// decodeAnchoringTx decodes an anchoring transaction.
+func (sbh *SubBridgeHandler) decodeAnchoringTx(data []byte) (common.Hash, *big.Int, error) {
+	anchoringData := new(types.AnchoringData)
+	if err := rlp.DecodeBytes(data, anchoringData); err != nil {
+		anchoringDataNoType := new(types.AnchoringDataInternalNoType)
+		if err := rlp.DecodeBytes(data, anchoringDataNoType); err != nil {
+			return common.Hash{}, nil, err
+		}
+		logger.Trace("decoded type0 anchoring tx", "blockNum", anchoringDataNoType.BlockNumber.String(), "blockHash", anchoringDataNoType.BlockHash.String(), "txHash", anchoringDataNoType.TxHash.String())
```

Comment:
The data is decoded as `types.AnchoringDataInternalNoType` type, but the log says "decoded type0 anchoring tx".

Reply:
Fixed.